xbmc/pvr/epg/Epg.h

```diff
@@ -217,11 +217,11 @@ namespace PVR
     bool QueuePersistQuery(const std::shared_ptr<CPVREpgDatabase>& database);
 
     /*!
-     * @brief Delete this table from the given database
+     * @brief Write the delete queries into the given database's queue
      * @param database The database.
-     * @return True if the table was deleted, false otherwise.
+     * @return True on success, false otherwise.
      */
-    bool Delete(const std::shared_ptr<CPVREpgDatabase>& database);
+    bool QueueDelete(const std::shared_ptr<CPVREpgDatabase>& database);
```

Comment:
I would name this method `QueueDeleteQueries`.

Reply:
Done